packages/openapi-v3/src/__tests__/unit/decorators/tags.decorator.unit.ts

```diff
@@ -0,0 +1,141 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {anOpenApiSpec} from '@loopback/openapi-spec-builder';
+import {expect} from '@loopback/testlab';
+import {getControllerSpec} from '../../..';
+import {oas} from '../../../decorators';
+
+describe('@oas.tags decorator', () => {
+  context('Without a top-level @oas.api definition', () => {
+    it('Allows a class to not be decorated with @oas.tags at all', () => {
+      class MyController {
+        @oas.get('/greet')
```

Comment:
Should we keep using `@get` instead?

Reply:
For the purposes of the unit test, it doesn't matter- though I intend to write code with the namespace consistently.